src/OpenApi/Tests/Fixtures/Dummy.php

```diff
@@ -81,7 +83,7 @@ class Dummy
     #[ApiProperty(push: true)]
     public ?RelatedDummy $relatedDummy = null;
 
-    public iterable $relatedDummies;
+    public Collection $relatedDummies;
```

Comment:
Is the change needed here ?

I dont see add call.

Reply:
You're right, done!